bundlepackages/azure-dotnet.csv

```diff
@@ -153,7 +153,7 @@ azuresearchservice,Microsoft.Azure.Search.Service,
 azureservicebus-ms,Microsoft.Azure.ServiceBus,
 azureservicebuseps,Microsoft.Azure.ServiceBus.EventProcessorHost,
 azuresqldbelasticscaleclient,Microsoft.Azure.SqlDatabase.ElasticScale.Client,
-azuresearchdoc,[tfm=netstandard2.0]Azure.Search.Documents
+azuresearchdocuments,[tfm=netstandard2.0]Azure.Search.Documents
```

Comment:
Per checking with Scott, this is expected as we change from manually generated IDs to automatically generated IDs